### PR TITLE
Creature renames

### PR DESCRIPTION
**Pull request by rr-**
#### Checklist

- [x] I have read the [coding conventions](https://github.com/rr-/Tomb1Main/blob/master/CONTRIBUTING.md#coding-conventions)
- [x] I have added a changelog entry about what my pull request accomplishes, or it is an internal change

#### Description

An increment for #521

### CODE REVIEW

File: src/game/creature.h [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "global/const.h"
+#include "global/types.h"
+
+#define CREATURE_SHOOT_RANGE SQUARE(WALL_L * 7) // = 51380224
+#define CREATURE_SHOT_DAMAGE 50
```
Comment (walkawayy) [inline src/game/creature.h]: This used to be `PEOPLE_SHOT_DAMAGE`. I don't see this new or old name used anywhere. Is it used somewhere outside the scope of this PR? Just want to make sure we didn't miss something.

File: src/game/creature.c [hunk 2]
```diff
@@ -0,0 +1,695 @@
...
+
+    GAME_VECTOR target;
+    target.x = g_LaraItem->pos.x;
+    target.y = g_LaraItem->pos.y - STEP_L * 3;
+    target.z = g_LaraItem->pos.z;
+
+    return LOS_Check(&start, &target);
+}
+
+bool Creature_ShootAtLara(
```
Comment (walkawayy) [inline src/game/creature.c:670]: Not sure about this new name. The original name better implies to me that it's checking if Lara was hit by a shot.

File: src/game/creature.c [hunk 3]
```diff
@@ -0,0 +1,695 @@
...
+    }
+
+    if (item->room_number != room_num) {
+        Item_NewRoom(item_num, room_num);
+    }
+
+    return true;
+}
+
+bool Creature_IsTargetable(ITEM_INFO *item, AI_INFO *info)
```
Comment (walkawayy) [inline src/game/creature.c]: What about `Creature_IsLaraTargetable`? The current name to me makes me think the function checks if an enemy creature is targetable. When it's actually checking if the creature can target Lara.

File: src/game/creature.h (hunk 1 above)
Comment (rr-) [reply src/game/creature.h]: This is unused indeed, removed

File: src/game/creature.c (hunk 2 above)
Comment (rr-) [reply src/game/creature.c:670]: But it also has a side effect of producing SFX.

File: src/game/creature.c (hunk 3 above)
Comment (rr-) [reply src/game/creature.c]: Wouldn't it make more sense then for this to stay near Lara? Lara_IsTargetable?

File: src/game/creature.c (hunk 2 above)
Comment (rr-) [reply src/game/creature.c:670]: Updated as per our conversation on Discord

File: src/game/creature.c (hunk 3 above)
Comment (rr-) [reply src/game/creature.c]: Updated as per our conversation on Discord

**Review by walkawayy — APPROVED** [review]
LGTM.

**Pull request merged**